model/operation.cc

```diff
@@ -167,6 +169,42 @@ namespace minsky
     return std::max(1.0f,std::min(0.5f*iWidth()*z/std::max(l,r),0.5f*iHeight()*z/h));  
   }
   
+namespace
+{
+	
+    void drawResizeHandle(cairo_t* cairo, double x, double y, double sf, double angle)
+    {
+      cairo::CairoSave cs(cairo);
+      cairo_translate(cairo,x,y);
+      cairo_rotate(cairo,angle);      
+      cairo_scale(cairo,sf,sf);
+      cairo_move_to(cairo,-1,-.2);
+      cairo_line_to(cairo,-1,-1);
+      cairo_line_to(cairo,1,1);
+      cairo_line_to(cairo,1,0.2);
+      cairo_move_to(cairo,-1,-1);
+      cairo_line_to(cairo,-.2,-1);
+      cairo_move_to(cairo,.2,1);
+      cairo_line_to(cairo,1,1);
+    }
+}  
+  
```

Comment:
Is there a way of drying out this code so we can reuse the Item::drawResizeHandles code as much as possible?

We will probably be asked to render the resize handles differently, perhaps as small boxes.